src/components/createEventPageComponents/CreateEvent.jsx

```diff
@@ -1,24 +1,54 @@
 import { addDoc } from "firebase/firestore";
-import React from "react";
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+import { MultiSelect } from "react-multi-select-component";
 
+import "react-datepicker/dist/react-datepicker.css";
+
+import LocationInput from "../reusableComponents/LocationInput";
 import { eventsCollection } from "../../lib/firebase/controller";
+import { interestList } from "../../lib/interestsList";
 
 function CreateEvent() {
+    const [startDate, setStartDate] = useState(new Date());
+    const [selectedInterets, setSelectedInterets] = useState([]);
+    const [loca, setLoca] = useState(null);
+
+    const options = interestList.map((obj) => {
+        return {
+            label: `${obj.title}`,
```

Comment:
wasn't an id required ? in the map function ?

Reply:
i'm not sure i get your question, but in here i'm simply converting the options const which represent the array of interests objects that an event may belong to, into an array of object in which each have the properties of label and value, all to make the multiselect component functional.